### PR TITLE
Update to Bleach 2.0

### PR DESCRIPTION
**Pull request by willkg**
This updates Standup to use Bleach 2.0 and html5lib 0.999999999 (9 9s). It
switches to reusing a Cleaner and Linker which is an easy performance gain. It
also adds the nofollow linkify callback so that all status message links are
marked as "nofollow".

### CODE REVIEW

**Comment by willkg** [comment]
I was thinking we can turn most of the passes in htmlify into either one html5lib filter or multiple filters. The advantage to that is that they get to work on the token stream rather than a chunk of HTML which would make them more focused and possibly faster.

Additionally, we can trivially add some more linkify callbacks for converting url link text into shortened version--something asked for in issue #367.

Anyhow, Bleach 2.0 gives us a lot of easy enhancement possibilities.

**Comment by willkg** [comment]
@pmac Want to skim this? It's pretty benign--there isn't a lot of change.

File: standup/status/models.py [hunk 1]
```diff
@@ -30,6 +32,10 @@
 ])
 
 
+CLEANER = Cleaner(tags=[])
+LINKER = Linker(callbacks=[trim_urls, nofollow])
```
Comment (willkg) [inline standup/status/models.py]: Pretty sure we should have been doing "nofollow" all along. So I took the time in this PR to add that.

Comment (pmac) [inline standup/status/models.py]: Very much agreed.

**Pull request merged**